tests/components/filter/test_sensor.py

```diff
@@ -15,309 +16,307 @@
-                "homeassistant.components.history.get_last_state_changes",
-                return_value=fake_states,
-            ):
-                with assert_setup_component(1, "sensor"):
-                    assert setup_component(self.hass, "sensor", config)
-                    self.hass.block_till_done()
-
-                self.hass.block_till_done()
-                state = self.hass.states.get("sensor.test")
-                assert "18.0" == state.state
-
-    def test_outlier(self):
-        """Test if outlier filter works."""
-        filt = OutlierFilter(window_size=3, precision=2, entity=None, radius=4.0)
-        for state in self.values:
-            filtered = filt.filter_state(state)
-        assert 21 == filtered.state
-
-    def test_outlier_step(self):
-        """
-        Test step-change handling in outlier.
-
-        Test if outlier filter handles long-running step-changes correctly.
-        It should converge to no longer filter once just over half the
-        window_size is occupied by the new post step-change values.
-        """
-        filt = OutlierFilter(window_size=3, precision=2, entity=None, radius=1.1)
-        self.values[-1].state = 22
-        for state in self.values:
+            with assert_setup_component(1, "sensor"):
+                assert await async_setup_component(hass, "sensor", config)
+                await hass.async_block_till_done()
+
+            await hass.async_block_till_done()
+            state = hass.states.get("sensor.test")
+            assert "18.0" == state.state
+
+
+async def test_outlier(values):
+    """Test if outlier filter works."""
+    filt = OutlierFilter(window_size=3, precision=2, entity=None, radius=4.0)
+    for state in values:
+        filtered = filt.filter_state(state)
+    assert 21 == filtered.state
+
+
+def test_outlier_step(values):
+    """
+    Test step-change handling in outlier.
+
+    Test if outlier filter handles long-running step-changes correctly.
+    It should converge to no longer filter once just over half the
+    window_size is occupied by the new post step-change values.
+    """
+    filt = OutlierFilter(window_size=3, precision=2, entity=None, radius=1.1)
+    values[-1].state = 22
+    for state in values:
+        filtered = filt.filter_state(state)
+    assert 22 == filtered.state
+
+
+def test_initial_outlier(values):
+    """Test issue #13363."""
+    filt = OutlierFilter(window_size=3, precision=2, entity=None, radius=4.0)
+    out = ha.State("sensor.test_monitored", 4000)
+    for state in [out] + values:
+        filtered = filt.filter_state(state)
+    assert 21 == filtered.state
+
+
+def test_unknown_state_outlier(values):
+    """Test issue #32395."""
+    filt = OutlierFilter(window_size=3, precision=2, entity=None, radius=4.0)
+    out = ha.State("sensor.test_monitored", "unknown")
+    for state in [out] + values + [out]:
+        try:
             filtered = filt.filter_state(state)
-        assert 22 == filtered.state
```

Comment:
What happened to this line ?

Reply:
I think that the diff is misaligned here. This line is on 207 in the new version of the file.

Reply:
When using the hide whitespace setting in the review/files tab it's easier to see the diff.

This assertion is intact. :+1: